lib/config/index.js

```diff
@@ -24,11 +26,20 @@ async function parseConfigs(env, argv) {
   const cliConfig = cliParser.getConfig(argv);
   const envConfig = envParser.getConfig(env);
 
+  if (Array.isArray(fileConfig.repositories)) {
```

Comment:
Can't we just resolve `fileConfig`?

Reply:
Well no, because `repositories` is not handled for this inside `resolveConfigPresets`.

Reply:
Is that itself a bug then? I would expect that we do dig into `repositories` and any other array with this line: https://github.com/renovateapp/renovate/blob/101226db977d8fa85c50bcef78f18e39735f1081/lib/config/presets.js#L78